code/src/js components/Summary.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Button2} from "js components/Button2";
+import {Button3} from "js components/Button3";
+
+export const Summary = (props) => {
+return (
+<>
+  <section className="summary">
+      <h1>Hey {props.name}! We are going to deliver your {props.drink} tomorrow to {props.address}! Enjoy! If you would like to get your coffee delivery {props.answeramount} times per week, please subscribe.</h1>
```

Comment:
My recommendation here is to have the text as a` <p>`. Because you only use h1 for headings, and this is more like text-content. And then style them separate in the Header.css as h1 and p instead of baking them together under the header-class :)